custom/intrahealth/tests/reports/test_dashboard_1.py

```diff
@@ -25,7 +25,10 @@ def test_extract_value_from_report_table_row_value_input_dict(self):
         report_table = {
             'fix_column': False,
             'comment': 'test_comment',
-            'rows': [],
+            'rows': [[{'html': '0'}, {'html': '4'}, {'html': '8'}, {'html': '12'}],
+                     [{'html': '1'}, {'html': '5'}, {'html': '9'}, {'html': '13'}],
+                     [{'html': '2'}, {'html': '6'}, {'html': '10'}, {'html': '14'}],
+                     [{'html': '3'}, {'html': '7'}, {'html': '11'}, {'html': '15'}]],
```

Comment:
Do we want to check some example with funny characters like in the original PR `{u'style': u'', u'html': u'pas de donn\xe9es'}`

Reply:
those are way better names, thanks https://github.com/dimagi/commcare-hq/pull/23245/commits/6bfa1926fd18985c93518e56247dcff8c943dfdc